orchestrator/broadcaster.go

```diff
@@ -3,6 +3,8 @@ package orchestrator
 import (
 	"context"
 
+	types2 "github.com/celestiaorg/celestia-app/x/qgb/types"
```

Comment:
the standard thing is naming this `qgbtypes` instead of types2

Reply:
[545ccee](https://github.com/celestiaorg/orchestrator-relayer/pull/560/commits/545ccee787c8c2b2674975f2af895dd35c783f89)

Reply:
definitely not blocking, just for future reference, `celestiatypes` is better, but imo we should be specific with our imports to improve readability. for instance, what happens if we need to import `"github.com/celestiaorg/celestia-app/x/blob/types"` or  `"github.com/celestiaorg/celestia-core/types"` as well? personally I'm not a fan of naming everything types, but the sdk and tendermint don't agree lol

Reply:
yeah I'm 100% down for adding CI to look at import names. For packages with types I think we could define a linter that is generic enough to be useful. Not sure about other package names that frequently require renaming

Reply:
https://github.com/celestiaorg/orchestrator-relayer/issues/563